File: 250301/살 수 있는 물건 2/things-able-to-buy-2.py
N = int(input())

if 0 <= N <= 10000 :
    if N >= 3000 :
        print("book")
    elif 1000 <= N < 3000 :
        print("mask")
    elif 500 <= N < 1000 :
        print("pen")
    else :
        print("no")
else :
    print("error")

File: 250722/살 수 있는 물건 2/things-able-to-buy-2.swift
let n = Int(readLine()!)!

if 0 <= n && n <= 10000 {
    if n >= 3000 {
        print("book")
    }
    else if n >= 1000 {
        print("mask")
    }
    else if n >= 500 {
        print("pen")
    }
    else {
        print("no")
    }
}